### ISSUE TITLE
Ability to add multiple subscription names

### ISSUE THREAD

**Issue by jringeisen** [themsaid/nova-cashier-manager#19]
I currently offer two subscriptions, pro and annual. The config only allows for one subscription name. Can we add the ability to do the following:

`'subscription_names' => ['pro', 'annual']`

This doesn't currently work and I have no way of checking users subscriptions that have an annual plan.

**Comment by eberkund** [comment]
Yes, please!!!  I don't understand why this is a limitation in the first place? Wouldn't most people not want to hide any subscriptions?

Very useful addon btw 🥇

**Comment by jringeisen** [comment]
@eberkund I don't think this repo is being maintained. A pull request has been created to fix this issue but hasn't been merged.

**Comment by bilfeldt** [comment]
Wasn't this fixed in PR #30 @eberkund or are you referring to another PR?

**Comment by jringeisen** [comment]
@bilfeldt it was, but it hasn't been merged.
